Libplanet/Assets/FungibleAssetValue.cs

```diff
@@ -27,7 +29,14 @@ namespace Libplanet.Assets
         /// </summary>
         public readonly Currency Currency;
 
-        internal readonly BigInteger RawValue;
+        /// <summary>
+        /// The internal representation of the fungible asset.
+        /// </summary>
+        /// <remarks>
+        /// Since this is an internal representation, this does not guarantee forward compatibility.
+        /// Therefore, do not depend on this value for permanent uses but only for volatile data.
+        /// </remarks>
+        public readonly BigInteger RawValue;
```

Comment:
How can we restore `FungibleAssetValue` by this `RawValue`? (`FungibleAssetValue(Currency, BigInteger)`  seems still internal)

Reply:
Added: b577a6dd78e0a07140f96fac1c067cc6b87620cf.